org.sasview.sasview.yml

```diff
@@ -0,0 +1,59 @@
+id: org.sasview.sasview
+runtime: org.kde.Sdk
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: sasview
+finish-args:
+  - "--socket=fallback-x11"
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--filesystem=~/sasview.log"
+  - "--filesystem=~/.sasview"
+  - "--filesystem=~/.sasmodels"
+modules:
+  - shared-modules/glu/glu-9.json
+  - name: sasview
+    buildsystem: simple
+    build-commands:
+      - mkdir $FLATPAK_DEST/bin
+      - cp -r . $FLATPAK_DEST/bin/
+    sources:
+      - type: archive
+        url: https://github.com/SasView/sasview/releases/download/v6.0.0/SasView-6.0.0-Ubuntu-22.04.tar.gz
+        sha256: 585d45689c1e5e6ba6e7e972e3ab15f0b522b9bdf9f9b0b11c550b1064137ea9
+  - name: python-opengl
+    buildsystem: simple
+    build-commands:
+      - cp -r OpenGL $FLATPAK_DEST/bin/
+    sources:
+      - type: archive
+        url:  https://files.pythonhosted.org/packages/72/b6/970868d44b619292f1f54501923c69c9bd0ab1d2d44cf02590eac2706f4f/PyOpenGL-3.1.7.tar.gz
+        sha256: eef31a3888e6984fd4d8e6c9961b184c9813ca82604d37fe3da80eb000a76c86
+  - name: sasview-desktop
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/applications/
+      - cp org.sasview.sasview.desktop /app/share/applications/
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/SasView/sasview/2e519c48373a4ebe3266225c5fccb3870f6403b2/build_tools/application_metadata/org.sasview.sasview.desktop
+        sha256: 52102796f2010447b447cd37fe6246362cd91234f3c762c2b1c14771a451fa85
+  - name: sasview-logo
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/icons/hicolor/512x512/apps/
+      - cp org.sasview.sasview.png /app/share/icons/hicolor/512x512/apps/org.sasview.sasview.png
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/SasView/sasview/2e519c48373a4ebe3266225c5fccb3870f6403b2/build_tools/application_metadata/org.sasview.sasview.png
+        sha256: fa4e310c68c88ad7f6ebdd4cf4b4e91fac1b729c71624fe6c971970640e5766d
+  - name: sasview-metainfo
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/metainfo/
+      - cp org.sasview.sasview.metainfo.xml /app/share/metainfo/
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/SasView/sasview/2e519c48373a4ebe3266225c5fccb3870f6403b2/build_tools/application_metadata/org.sasview.sasview.metainfo.xml
+        sha256: 47f09768043057bf39b43e7162a21577f2e5404f5b9549eceae31291725c6ef4
```

Comment:
merge the sources with the sasview module and put the build commands as `post-install` there

Reply:
Thanks for this suggestion. I've just pushed a commit which does this.